### PR TITLE
Added adduser and remuser command to james!

### PR DESCRIPTION
**Pull request by Victonator**

### CODE REVIEW

File: pkg/commands/game/add_user.go [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+	"fmt"
+
+	"github.com/itfactory-tm/thomas-bot/pkg/command"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/itfactory-tm/thomas-bot/pkg/sudo"
+)
+
+// adduser contains the bob!adduser and bob!remuser command
+type AddUserCommand struct{}
```
Comment (meyskens) [inline pkg/commands/game/add_user.go:13]: Maybe this should be named UserCommand ?

File: pkg/commands/game/add_user.go [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+func (m *AddUserCommand) addUser(s *discordgo.Session, msg *discordgo.MessageCreate) {
+	if !(sudo.IsItfAdmin(msg.Author.ID) || sudo.IsAdmin(msg.Author.ID)) {
+		s.ChannelMessageSend(msg.ChannelID, fmt.Sprintf("%s is not in the sudoers file. This incident will be reported.", msg.Author.ID))
+		return
+	}
+
+	matched := msg.Message.Mentions
+	if len(matched) < 1 {
```
Comment (meyskens) [inline pkg/commands/game/add_user.go:33]: What if 2 users are specified? Maybe we should error in that case, or loop over them

File: pkg/commands/game/add_user.go [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+		s.ChannelMessageSend(msg.ChannelID, "You need to specify a user")
+		return
+	}
+	user := matched[0].ID
+
+	roles, err := s.GuildRoles(msg.GuildID)
+	if err != nil {
+		s.ChannelMessageSend(msg.ChannelID, fmt.Sprintf("Error: %v", err))
```
Comment (meyskens) [inline pkg/commands/game/add_user.go:41]:
```suggestion
		s.ChannelMessageSend(msg.ChannelID, fmt.Sprintf("Error getting guild roles: %v", err))
```

File: pkg/commands/game/add_user.go [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+	s.ChannelMessageSend(msg.ChannelID, ("User removed! <@" + user + ">"))
+}
+
+// Info return the commands in this package
+func (m *AddUserCommand) Info() []command.Command {
+	return []command.Command{
+		command.Command{
+			Name:        "adduser",
+			Category:    command.CategoryModeratie,
+			Description: "Add a user to the ITF Gamer role (ITF admin only)",
```
Comment (meyskens) [inline pkg/commands/game/add_user.go:103]: ITF Admin sounds confusing... should this better be ITF Game Admin?

File: pkg/commands/game/add_user.go [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+// Register registers the handlers
+func (m *AddUserCommand) Register(registry command.Registry, server command.Server) {
+	registry.RegisterMessageCreateHandler("adduser", m.addUser)
+	registry.RegisterMessageCreateHandler("remuser", m.remUser)
+}
+
+func (m *AddUserCommand) addUser(s *discordgo.Session, msg *discordgo.MessageCreate) {
+	if !(sudo.IsItfAdmin(msg.Author.ID) || sudo.IsAdmin(msg.Author.ID)) {
```
Comment (meyskens) [inline pkg/commands/game/add_user.go:27]: Think we can better only do gamer admin

**Comment by meyskens** [comment]
Looks great! Just a few small things then happy to merge it in!

File: pkg/commands/game/add_user.go (hunk 2 above)
Comment (Victonator) [reply pkg/commands/game/add_user.go:33]: It selects the first mentionned user, but looping over multiple users sounds useful!

**Comment by Victonator** [comment]
Will do!

File: pkg/commands/game/add_user.go (hunk 2 above)
Comment (Victonator) [reply pkg/commands/game/add_user.go:33]: I'll also change matched to mention as matched doesn't really make sense in this context

**Comment by meyskens** [comment]
LGTM

**Pull request merged**